### PR TITLE
Fix S3878 warning

### PR DESCRIPTION
**Pull request by Zombach**
# Pull Request

## The issue or feature being addressed

[#1290](https://github.com/App-vNext/Polly/issues/1290)

## Details on the issue fix or feature implementation

 * [x]  Suppress [S3878](https://rules.sonarsource.com/csharp/RSPEC-3878/) in the code or fix the warning

## Confirm the following

- [x]  I started this PR by branching from the head of the default branch
- [x]  I have targeted the PR to merge into the default branch
- [x]  I have included unit tests for the issue/feature
- [ ]  I have successfully run a local build

### CODE REVIEW

File: test/Polly.Specs/Wrap/PolicyWrapSpecs.cs [hunk 1]
```diff
@@ -308,7 +311,7 @@ public void Wrapping_nothing_using_static_wrap_strongly_typed_syntax_should_thro
     public void Wrapping_only_one_policy_using_static_wrap_strongly_typed_syntax_should_throw()
     {
         Policy<int> singlePolicy = Policy<int>.Handle<Exception>().Retry();
-        Action config = () => Policy.Wrap<int>(new[] { singlePolicy });
+        Action config = () => Policy.Wrap<int>(singlePolicy);
```
Comment (martincostello) [inline test/Polly.Specs/Wrap/PolicyWrapSpecs.cs:314]: Was this using an array to call a specific overload?

File: test/Polly.Specs/Wrap/PolicyWrapSpecsAsync.cs [hunk 2]
```diff
@@ -253,7 +253,7 @@ public void Wrapping_nothing_using_static_wrap_syntax_should_throw()
     public void Wrapping_only_one_policy_using_static_wrap_syntax_should_throw()
     {
         AsyncPolicy singlePolicy = Policy.Handle<Exception>().RetryAsync();
-        Action config = () => Policy.WrapAsync(new[] { singlePolicy });
+        Action config = () => Policy.WrapAsync(singlePolicy);
```
Comment (martincostello) [inline test/Polly.Specs/Wrap/PolicyWrapSpecsAsync.cs:256]: Same question here.

File: test/Polly.Specs/Wrap/PolicyWrapSpecsAsync.cs [hunk 3]
```diff
@@ -308,7 +308,7 @@ public void Wrapping_nothing_using_static_wrap_strongly_typed_syntax_should_thro
     public void Wrapping_only_one_policy_using_static_wrap_strongly_typed_syntax_should_throw()
     {
         AsyncPolicy<int> singlePolicy = Policy<int>.Handle<Exception>().RetryAsync();
-        Action config = () => Policy.WrapAsync<int>(new[] { singlePolicy });
+        Action config = () => Policy.WrapAsync<int>(singlePolicy);
```
Comment (martincostello) [inline test/Polly.Specs/Wrap/PolicyWrapSpecsAsync.cs:311]: +1

Comment (Zombach) [reply test/Polly.Specs/Wrap/PolicyWrapSpecsAsync.cs:311]: image

File: test/Polly.Specs/Wrap/PolicyWrapSpecsAsync.cs (hunk 2 above)
Comment (Zombach) [reply test/Polly.Specs/Wrap/PolicyWrapSpecsAsync.cs:256]: image

File: test/Polly.Specs/Wrap/PolicyWrapSpecs.cs (hunk 1 above)
Comment (Zombach) [reply test/Polly.Specs/Wrap/PolicyWrapSpecs.cs:314]: image

**Review by martincostello — APPROVED** [review]

**Pull request merged**